generate/writeToJsonFile.js

```diff
@@ -0,0 +1,45 @@
+import path from 'path'
+import fs from 'fs'
+
+const JsonStringifyParameters = {
+  replacer: null,
+  space: 4,
```

Comment:
Sorry I did not understand what were these 2. 
Feel free to tell me after latest PR update.